app/furniture/marketplace/product.rb

```diff
@@ -44,6 +44,8 @@ class Product < Record
     scope :with_menu_tag, -> { joins(:tags).merge(Tag.menu_tag) }
     scope :without_menu_tag, -> { where.not(id: with_menu_tag) }
     scope :sort_alpha, -> { order(name: :asc) }
+    scope :menu_tag_product, -> { with_all_rich_text.merge(unarchived).merge(sort_alpha) }
+    scope :menu_product, -> { with_all_rich_text.merge(unarchived).merge(without_menu_tag).merge(sort_alpha) }
```

Comment:
A few things about these two new scopes raise questions for me:
* why are you `merge`ing scopes from the same model? i don't think you need nor want to do that; you can just do `{... unarchived.without_menu_tag.sort_alpha ...}`
* do we want to encode eager-loading as part of a scope? we don't typically do that, because it makes the scope only suitable for that one particular use case, and bloated for other purposes; in this example, if you wanted to count the menu_products, this scope would be unnecessarily eager-loading all rich text in order to do that count
* these are scopes on a Product model, so adding `product` to the scope names seems redundant (i.e. `Product.menu` is a better scope name than `Product.menu_product`)
* it seems like the `menu_tag_product` scope is defined as a superset of the `menu_product`, which doesn't seem right

Reply:
> do we want to encode eager-loading as part of a scope

I think this makes sense to do now that I've moved the scope to methods on the component